src/test.js

```diff
@@ -491,6 +493,13 @@ function synchronize( callback, priority ) {
 
 	if ( priority ) {
 		config.queue.splice( priorityCount++, 0, callback );
+	} else if ( random ) {
+		if ( !randomNumber ) {
+			randomNumber = randomNumberGenerator( random );
```

Comment:
I'm not a fan of these misleading names. What about `seededUnitSample = unitSamplerGenerator( random );`?